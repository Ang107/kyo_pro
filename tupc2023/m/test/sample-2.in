6
30594 32322 46262
63608 59020 98436
90150 32740 67209
82886 4627 54813
3112 67989 74995
60872 9967 9051
