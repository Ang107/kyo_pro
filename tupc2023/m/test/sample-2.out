715162883
838096208
930330061
405079896
880764907
526006962
